composer.json

```diff
@@ -62,7 +62,7 @@
         "symfony/css-selector": "^2.7|^3.0|^4.0",
         "phpoption/phpoption": "^1.1",
         "jms/serializer-bundle": "^2.0|^3.0",
-        "jms/serializer": "^1.13|^2.0",
+        "jms/serializer": "^1.5|^2.0",
```

Comment:
this change needs to be reverted (see the failing test)

Reply:
I was trying to fix #1961 because FOSRestBundle allows installing `jms/serializer <1.12` and triggers an error. Should we drop support for `<1.12` by adding it to `conflicts` section?

Reply:
Yes, I think that would be better.